spec/models/goods_nomenclature_spec.rb

```diff
@@ -92,4 +92,72 @@
       end
     end
   end
+
+  describe '#rules_of_origin' do
+    before do
+      # Needed because goods nomenclature isn't declarable but #rules_of_origin
```

Comment:
I think we can reflect that section, chapters, etc, all have declarable and just treat them as ducks/trust the data

Reply:
Ok, will change `Chapter` to always return false for `#declarable?`

Reply:
This override will still be required though because we want to bypass the short-circuit